pytest_mpl/plugin.py

```diff
@@ -131,7 +144,7 @@ def item_function_wrapper(*args, **kwargs):
 
             baseline_remote = baseline_dir.startswith('http')
 
-            with plt.style.context(style):
+            with plt.style.context(style), switch_backend(backend):
```

Comment:
Now 2.0 is out, I'd be fine with dropping 1.4 (I normally like keeping compatibility with the latest two versions)